schema/schema.sql

```diff
@@ -0,0 +1,84 @@
+DROP TABLE IF EXISTS track_tag_assignment;
+DROP TABLE IF EXISTS album_tag_assignment;
+DROP TABLE IF EXISTS tag;
+DROP TABLE IF EXISTS track_purchase;
+DROP TABLE IF EXISTS album_purchase;
+DROP TABLE IF EXISTS country;
+DROP TABLE IF EXISTS track;
+DROP TABLE IF EXISTS album;
+DROP TABLE IF EXISTS artist;
+DROP TABLE IF EXISTS subscriber;
+
+
+CREATE TABLE subscriber (
+    subscriber_id SMALLINT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+    email VARCHAR(255) UNIQUE NOT NULL,
+    name VARCHAR(255) NOT NULL
+);
+
+CREATE TABLE artist (
+    artist_id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+    name VARCHAR(255) NOT NULL,
+    url VARCHAR(255) UNIQUE NOT NULL
+);
+
+CREATE TABLE album (
+    album_id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+    title VARCHAR(255) NOT NULL,
+    artist_id INT NOT NULL,
+    FOREIGN KEY (artist_id) REFERENCES artist(artist_id)
+);
+
+CREATE TABLE track (
+    track_id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+    title VARCHAR(255) NOT NULL,
```

Comment:
Some tracks can actually have titles longer than 255 characters, maybe text would be better?

Reply:
Note, this also possibly affects album title and artist name.

Reply:
Done!